RecoLocalMuon/CSCSegment/src/CSCSegAlgoRU.cc

```diff
@@ -560,7 +560,6 @@ float CSCSegAlgoRU::fit_r_phi(const AlgoState& aState, const SVector6& points, i
 
 void CSCSegAlgoRU::baseline(AlgoState& aState, int n_seg_min) const {
   int nhits = aState.proto_segment.size();
-  ChamberHitContainer::const_iterator iRH_worst;
```

Comment:
this is already done in #25684 

to avoid a need to rebase there may I propose to drop this PR.

Reply:
I'll close this then.